manifests/charts/istio-operator/values.yaml

```diff
@@ -24,3 +24,7 @@ operator:
     requests:
       cpu: 50m
       memory: 128Mi
+
+# Additional labels and annotations to apply on the pod level for monitoring and logging configuration.
+operatorLabels: {}
```

Comment:
This doesn't not follow standard helm practices. There is no need to prefix with `operator` - its already in the operator chart, its redundant.

Reply:
We should be consistent with the `selectorLabels` (although I have concerns with that as well).

Reply:
Renamed